cortex_xdr/plugin.spec.yaml

```diff
@@ -0,0 +1,232 @@
+        required: false
+    domain:
+        title: "Domain"
+        type: string
+        description: "Domain"
+        required: false
+    endpoint_id:
+        title: "Endpoint ID"
+        type: string
+        description: "Endpoint iD"
+        required: false
+    endpoint_name:
+        title: "Endpoint Name"
+        type: string
+        description: "Endpoint name"
+        required: false
+    endpoint_status:
+        title: "Endpoint Status"
+        type: string
+        description: "Endpoint status"
+        required: false
+    endpoint_type:
+        title: "Endpoint Type"
+        type: string
+        description: "Endpoint type"
+        required: false
+    endpoint_version:
+        title: "Endpoint Version"
+        type: string
+        description: "Endpoint version"
+        required: false
+    first_seen:
+        title: "First Seen"
+        type: integer
+        description: "First seen"
+        required: false
+    install_date:
+        title: "Install Date"
+        type: integer
+        description: "Install date"
+        required: false
+    installation_package:
+        title: "Installation Package"
+        type: string
+        description: "Installation package"
+        required: false
+    ip:
+        title: "IP"
+        type: "[]string"
+        description: "IP"
+        required: false
+    is_isolated:
+        title: "Is Isolated"
+        type: string
+        description: "Is isolated"
+        required: false
+    last_seen:
+        title: "Last Seen"
+        type: integer
+        description: "Last seen"
+        required: false
+    operational_status:
+        title: "Operational Status"
+        type: string
+        description: "Operational status"
+        required: false
+    operational_status_description:
+        title: "Operational Status Description"
+        type: string
+        description: "Operational status description"
+        required: false
+    os_type:
+        title: "OS Type"
+        type: string
+        description: "OS type"
+        required: false
+    scan_status:
+        title: "Scan Status"
+        type: string
+        description: "Scan status"
+        required: false
+    users:
+        title: "Users"
+        type: "[]string"
+        description: "Users"
+        required: false
+
+connection:
+  fqdn:
+    title: FQDN
+    description: Fully qualified domain name
+    type: string
+    required: true
+    example: https://api-example.xdr.us.paloaltonetworks.com/
+  api_key_id:
+    title: API Key ID
+    description: The API Key ID shown in the API Keys table in settings. e.g. 1, 2, 3
+    type: int
+    required: true
+    example: 1
+  api_key:
+    title: API Key
+    description: The API Key that is generated when creating a new key
+    type: credential_secret_key
+    required: true
+    example: '1234123412341234asdfasdfasdfasdfasdf1234123412341234123412341234asdfasdfasdfasdfasdf123412341234123412341234asdfasdfasdfasdfasdf'
```

Comment:
Use our example API key

Reply:
Disagree...This is both obviously fake, and kind of resembles what Cortex generates. I'd like to stick with this so it's not confused with anything else.